searchlib/src/vespa/searchlib/queryeval/blueprint.cpp

```diff
@@ -308,6 +317,13 @@ IntermediateBlueprint::calculateState() const
     return state;
 }
 
+ExecuteInfo
+IntermediateBlueprint::computeNext(const Blueprint & child, const ExecuteInfo & execInfo) const
```

Comment:
since strictness is handled externally, maybe this function should only be about hit ratios?

Reply:
Fixed